MeetupPlatformApi/Features/Authentication/RefreshTokenPair/RefreshTokenPairFeature.cs

```diff
@@ -0,0 +1,62 @@
+﻿namespace MeetupPlatformApi.Features.Authentication.RefreshTokenPair;
+
+using AutoMapper;
+using MeetupPlatformApi.Authentication.Helpers;
+using MeetupPlatformApi.Domain;
+using MeetupPlatformApi.Persistence.Context;
+using MeetupPlatformApi.Seedwork.WebApi;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+[ApiSection(ApiSections.Authentication)]
+public class RefreshTokenPairFeature : FeatureBase
+{
+    private readonly ApplicationContext context;
+    private readonly IMapper mapper;
+    private readonly TokenHelper tokenHelper;
+
+    public RefreshTokenPairFeature(ApplicationContext context, IMapper mapper, TokenHelper tokenHelper)
+    {
+        this.context = context;
+        this.mapper = mapper;
+        this.tokenHelper = tokenHelper;
+    }
+
+    [HttpPost("/api/users/refresh-tokens")]
+    public async Task<IActionResult> RefreshTokenPair([FromBody] string refreshToken)
+    {
+        var refreshTokenExpires = tokenHelper.GetExpires(refreshToken);
+        if (DateTime.UtcNow > refreshTokenExpires)
```

Comment:
`JwtBearer` has default mechanisms for parsing tokens and ensuring that they are not expired. So, please don't use custom solutions if we have a built-in functionality that can fully satisfy our needs.